docs/tutorials/getting-started.ipynb

```diff
@@ -0,0 +1,469 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "7def9ef3-b088-46ca-8733-18771cad4028",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from scipp.constants import physical_constants\n",
+    "\n",
+    "G = physical_constants('Newtonian constant of gravitation', with_variance=True)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "fbe9aa36-dd99-48aa-80e6-567e1b658a36",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "m1 = sc.scalar(1.0, variance=1e-9, unit='kg')\n",
+    "m2 = sc.scalar(2.0, variance=1e-9, unit='kg')\n",
+    "G * m1 * m2 / (length.to(unit='m') ** 2)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6f988a7e-8bde-4847-b166-c6b2e21cb3b9",
+   "metadata": {},
+   "source": [
+    "## Arrays\n",
+    "\n",
+    "So far we have worked with single-valued quantities (indicated by an empty shape tuple `()` above) represented by a `scipp.Variable`.\n",
+    "The values (and variances) of a `scipp.Variable` work much like a NumPy array.\n",
+    "Consider a 2-D NumPy array:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "9adc7236-611b-42b7-95bf-21651fa1ebcb",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "\n",
+    "values = np.random.rand(3, 3)\n",
+    "values"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3432055c-9deb-4b4c-ab31-327cf9d6f153",
+   "metadata": {},
+   "source": [
+    "We can create a `scipp.Variable` from this using `array`:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "029ab90c-d504-4b4f-8764-3dc6abf7123d",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "var = sc.array(dims=['location', 'time'], values=values, unit='K')\n",
+    "var"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "7c4f6b57-47d0-4d35-bf98-8aae7a38bfa0",
+   "metadata": {},
+   "source": [
+    "Conceptually this is an array-valued quantity.\n",
+    "The `unit` property was discussed above.\n",
+    "The new element here is the `dims` property, which assigns labels to the axes of the NumPy array.\n",
+    "With a plain NumPy array it is hard to tell what the following operation does (and whether it matches the intention of the author):"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "f40062fd-3a6a-49f3-a682-35ca8a49253f",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "values[2] + values[:, 2]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "47c784fc-5898-427a-9e4f-14791b970d5d",
+   "metadata": {},
+   "source": [
+    "Scipp's dimension labels are specified when slicing and make this clear and unambiguous:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "0fddf047-538e-42b0-9965-59e2722ad067",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "var['time', 2] + var['location', 2]"
```

Comment:
I think automatic broadcasting is a bit too much to have in such a low-level tutorial. Maybe just extracting a single column is enough?

Reply:
I wanted to illustrate how dimension labels avoid pitfalls of NumPy. Do you have an alternative suggestion?